test/spacesuit_auth_middleware_test.exs

```diff
@@ -1,91 +1,167 @@
 defmodule SpacesuitAuthMiddlewareTest do
   use ExUnit.Case
+  import Mox
```

Comment:
importing gives us `mock` and `stub` as local functions. If you prefer it to be more explicit, you can `require Mox` and the calls will be `Mox.mock` and `Mox.stub`.